package.json

```diff
@@ -40,7 +40,7 @@
   "scripts": {
     "compile": "tsc",
     "test": "tsc && nyc mocha",
-    "coverage": "nyc mocha --reporter=json > coverage.json && codecov -f ./coverage.json --disable=gcov",
+    "coverage": "nyc mocha --reporter=cobertura && codecov --disable=gcov",
```

Comment:
`"coverage": "nyc --reporter=cobertura mocha  && codecov --disable=gcov",`

Give this a try?